x-pack/plugins/session_view/public/components/session_view_search_bar/index.tsx

```diff
@@ -53,7 +53,7 @@ export const SessionViewSearchBar = ({
   return (
     <div data-test-subj="sessionView:searchInput" css={{ position: 'relative' }}>
       <EuiSearchBar query={searchQuery} onChange={onSearch} />
-      {showPagination && (
+      {!!showPagination && (
```

Comment:
maybe the !! should go above, e.g  const showPagination = !!searchResults?.length

Reply:
Good catch btw